app/build.gradle

```diff
@@ -67,9 +67,15 @@ dependencies {
     implementation 'com.google.firebase:firebase-auth-ktx'
     implementation 'com.google.firebase:firebase-storage-ktx'
 
+    // Mockito library for testing
+    testImplementation 'org.mockito:mockito-inline:5.2.0'
+    androidTestImplementation 'org.mockito:mockito-core:5.2.0'
+    androidTestImplementation 'org.mockito.kotlin:mockito-kotlin:4.1.0'
+    androidTestImplementation 'org.mockito:mockito-android:5.2.0'
```

Comment:
If we end up not using mock testing in [PostTest](https://github.com/SDP-BeGreen/BeGreen/blob/af44248df800de0b0205f4ba0763b57673f6ee9c/app/src/test/java/com/github/sdp_begreen/begreen/models/PostTest.kt), I think it would be better to remove those dependency, no to have unused dependency in our gradle. We can always add them later when we need them.